<launch>

  <!-- 使用rosbag时, 必须设置为true; 实物实验请设置为false -->
  <param name="/use_sim_time" value="true"/>

  <!-- 启动障碍物管理 -->
  <node pkg="dynamic_perception" type="obs_manager_node" name="obs_manager_node" output="screen">
    <remap from="/trajs_predicted" to="/obstacle_prediction_node/obstacle_prediction/trajs_predicted"/>
    <param name="/obs_manager/use_GroundTruth" value="false" type="bool"/>
  </node>

</launch>
